### ISSUE TITLE
📝 Correction | Cloudflare Privacy Policy Link Outdated

### ISSUE THREAD

**Issue by ghost** [privacytools/privacytools.io#2016]
## Description
Cloudflare dns privacy policy link is outdated

## Why I am making the suggestion
Outdated info

I did post this in the forum but none of the devs fixed it
## My connection with the software
None

- [ ] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by Mikaela** [comment]
[The forum thread has a bit more details](https://forum.privacytools.io/t/cloudflare-changed-doh-privacy-policy/3961?u=mikaela)

> They will now log users “anonymized” ip address and the log will be kept 25h and much more none privacy friendly stuff that i did not mentioned https://developers.cloudflare.com/1.1.1.1/privacy/public-dns-resolver/ 11

**Comment by jackmchou** [comment]
How should this be updated? 

Our link in the DNS providers links to Cloudflare's [privacy policy](https://www.cloudflare.com/privacypolicy/) page, which in turn has the [link](https://developers.cloudflare.com/1.1.1.1/privacy/public-dns-resolver/) mentioned in the forum thread. It's under item 2 information we collect -> Public DNS Resolver Users

**Comment by jonaharagon** [comment]
The privacy policy we currently link to seems to cover the 1.1.1.1 service.

**Comment by Mikaela** [comment]
> `"We will collect limited DNS query data that is sent to the resolvers. This data does not contain user IP addresses or any other personally identifiable information, and the bulk of the data is only stored for 24 hours."' href="https://developers.cloudflare.com/1.1.1.1/commitment-to-privacy/privacy-policy/privacy-policy/">Some</a>`

Is what the link and tooltip currently say, that is a 404 error and the quote is outdated at least on the hours, as they currently say:

> [...] We will collect limited DNS query data that is sent to our 1.1.1.1 resolver. Our 1.1.1.1 resolver service does not log personal data, and the bulk of the limited non-personally identifiable query data is only stored for 25 hours. [...]

on https://www.cloudflare.com/privacypolicy/

**Comment by jonaharagon** [comment]
You're right. For my own future reference, this issue is referring to the link/tooltip under 'logging' and not the 'privacy policy' link as was suggested. This does need to be fixed.

![image](https://user-images.githubusercontent.com/3637842/89986476-3eeea180-dc42-11ea-8615-9f4df8981b79.png)
